crossing-repair.lic

```diff
@@ -12,12 +12,22 @@ class CrossingRepair
   def initialize
     setup
 
-    @repair_info.each do |repairer, items|
```

Comment:
It looks like your other change got included here. Can you remove the `crossing-repair` changes from this PR?